tests/fun3d_tests/fun3d_aero_remesh/run_fun3d_analysis.py

```diff
@@ -28,7 +28,7 @@
 model = FUNtoFEMmodel("wing")
 wing = Body.aeroelastic("wing", boundary=2)
 wing.register_to(model)
-test_scenario = Scenario.steady("turbulent", steps=5000).set_temperature(
+test_scenario = Scenario.steady("euler", steps=5000).set_temperature(
```

Comment:
Are you sure you want to switch this to Euler?

Reply:
This was already Euler I was just using a mismatching scenario folder called turbulent.